src/stc/ScintillaWX.cpp

```diff
@@ -844,6 +845,31 @@ sptr_t ScintillaWX::WndProc(unsigned int iMessage, uptr_t wParam, sptr_t lParam)
       case SCI_GETDIRECTPOINTER:
             return reinterpret_cast<sptr_t>(this);
 
+#ifdef __WXMSW__
+      // ScintillaWin
+      case WM_IME_STARTCOMPOSITION: 	// dbcs
+          // TODO: We haven't ported WM_IME_COMPOSITION yet, so always use windowed IME to make it at least work
+          //if (KoreanIME() || imeInteraction == imeInline) {
+          if (false) {
+              return 0;
+          }
+          else {
+              ImeStartComposition();
+              return stc->wxControl::MSWWindowProc(iMessage, wParam, lParam);
+          }
+
+      case WM_IME_ENDCOMPOSITION: 	// dbcs
+          ImeEndComposition();
+          return stc->wxControl::MSWWindowProc(iMessage, wParam, lParam);
+
+      case WM_IME_KEYDOWN:
+      case WM_IME_REQUEST:
+      case WM_IME_COMPOSITION:
+      case WM_IME_SETCONTEXT:
+          // TODO: Port event handling from ScintillaWin
```

Comment:
I'd like to understand what exactly is meant here. Is it for inline IME support? If so, we should probably say so directly and maybe remove "TODO" as we have no plans for doing it at the moment (and just say that we'd need to implement this for inline IME).

Reply:
Yes, it is for inline IME